sqlanalytics/resource_visualization.go

```diff
@@ -0,0 +1,195 @@
+package sqlanalytics
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"strconv"
+	"strings"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/databrickslabs/terraform-provider-databricks/sqlanalytics/api"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Visualization ...
+type Visualization struct {
+	ID          string `json:"id,omitempty" tf:"computed"`
+	QueryID     string `json:"query_id"`
+	Type        string `json:"type"`
+	Name        string `json:"name"`
+	Description string `json:"description,omitempty"`
+	Options     string `json:"options"`
+}
+
+type visualizationResource struct {
```

Comment:
to be consistent with the rest of codebase, remove this struct and inline methods